apps/design-system-docs/src/components/link-card/style.module.scss

```diff
@@ -0,0 +1,33 @@
+@use '@wordpress/base-styles/colors';
+@use '@wordpress/base-styles/variables';
+
+.link-card {
+	position: relative;
+	transition: box-shadow 0.2s ease;
+	
+	&:hover {
+		box-shadow: 0 0 0 1px colors.$gray-900;
+	}
```

Comment:
@fcoveram For the hover style, I did a black border as suggested by @jasmussen in p1747927686862889/1747926546.378209-slack-CNGQYA3B9, and added a slight transition. Let me know if you want something different.

https://github.com/user-attachments/assets/3a45a4ac-6682-4e50-acb1-483663c12582

Reply:
Should these be DS components :)

Reply:
We'll add it as a pattern, at the very least!

Reply:
For me a pattern shouldn't have custom CSS, and this has a lot.